### PR TITLE
11472 tobacco speculation vacancy tax

### PR DESCRIPTION
**Pull request by garyfan1**
*Issue #:* /bcgov/entity#11472

*Description of changes:*
Add Tobacco Tax Act and Speculation Vacancy Tax Act to Registration, applying rules that were applied to Carbon Tax Act.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ppr-ui",
-  "version": "0.3.6",
+  "version": "0.3.7",
```
Comment (cameron-eyds) [inline ppr-ui/package.json:3]: Nice :)

**Review by cameron-eyds — COMMENTED** [review]
Code looks good!

Just a quick question, it appears you've added selectable options to the Registrations Drop Down? Is that correct?

Are these flows supported within the current architecture? Or is this all that is required to support new Registration types?

**Comment by garyfan1** [comment]
The UI flow would be the same as Carbon Tax Act, so I believe it is fully supported until clicking that "submit registration" button; which the backend would complain about not recognizing Tobacco/Speculation because it has not been implemented yet.
API ticket is bcgov/entity#11471
Backend response when trying to submit a Tobacco Tax Act Registration:
image

**Comment by cameron-eyds** [comment]
> The UI flow would be the same as Carbon Tax Act, so I believe it is fully supported until clicking that "submit registration" button; which the backend would complain about not recognizing Tobacco/Speculation because it has not been implemented yet. API ticket is [bcgov/entity#11471](https://github.com/bcgov/entity/issues/11471) Backend response when trying to submit a Tobacco Tax Act Registration: image

Sounds good, thanks for the explanation!

**Review by cameron-eyds — APPROVED** [review]

**Review by doug-lovett — APPROVED** [review]
Looks good.

**Pull request merged**